### ISSUE TITLE
Translations of ipython core project website

### ISSUE THREAD

**Issue by steppi** [ipython/ipython#14442]
Hi IPython team,

With support from the CZI [Scientific Python Community and Communications Infrastructure](https://scientific-python.org/doc/scientific-python-community-and-communications-infrastructure-2022.pdf) grant, [Quansight Labs](https://labs.quansight.org/) is able to offer assistance with developing and publishing translations of the brochure websites for the [Scientific Python Core Project](https://scientific-python.org/specs/core-projects/). You may have seen that translations into some languages are already available for https://numpy.org/, with a version switcher in the top right corner; we're offering to help core projects integrate something similar into their websites. Our aim is to accomplish this in a way that requires minimal effort from the core project maintainers. 

I've been tasked with setting up the software infrastructure for translations, and am posting issues today to ask core project teams if they would like to participate. I've published an FAQ here: https://scientific-python-translations.github.io/faq/, with more information. Please take a look and let me know if you have any questions. If you decide to participate, I can begin setting up the translation infrastructure, which will require no work or input from maintainer teams (See [here](https://scientific-python-translations.github.io/faq/#what-work-is-expected-from-project-maintainers) in the FAQ for more details on the process).

As a disclaimer, @Carreau also works at Quansight Labs and I briefly discussed this translation project with him last year.

**Comment by Carreau** [comment]
Hey, 

I think those days in this project it's mostly me.

I'm happy to help set up translations, but my main concern is that I believe the docs themselves need a rewrite/overhaul as they have been rotting for a few years.

**Comment by steppi** [comment]
Hey @Carreau. Yeah, it does seem like it would make sense to do a rewrite/overhaul first. We'd just be having the brochure website ipython.org translated, so I wonder if it makes sense to move ipython.org to the [Scientific Python Hugo Theme](https://github.com/scientific-python/scientific-python-hugo-theme) and flesh it out a little more so that it contains more info that would be helpful in translation. It doesn't seem like it would be too much work, so I think I could help out with that.

**Comment by Carreau** [comment]
The source of ipython.org is at https://github.com/ipython/ipython-website, I do not have any objections, but I think a revamp of the website as well should likely go through the jupyter governance and get some of it's own funding as well to clean it up.

seeing how the revamp of the Jupyter website wend before I'm not sure I have the strenght to go through that. Are you coming the retreat in seattle ? If so we can talk about that one on one.

**Comment by steppi** [comment]
> seeing how the revamp of the Jupyter website wend before I'm not sure I have the strenght to go through that. Are you coming the retreat in seattle ? If so we can talk about that one on one.

Yes, I'll be at the retreat. Sounds good. I was also thinking of creating an issue on the project board to discuss translations, but it looks like there's already a lot of issues there, so I'm not sure about it.

**Comment by Carreau** [comment]
You can still create an issue, and we can not talk about it :-)

**Comment by steppi** [comment]
Issue created here, https://github.com/scientific-python/summit-2024/issues/20
